src/goto-bmc/module_dependencies.txt

```diff
@@ -0,0 +1,3 @@
+util
+libcprover-cpp
+goto-programs
```

Comment:
Why this dependency? Shouldn't goto-bmc only depend on libcprover-cpp?

Reply:
We need this because of the check inside `src/goto-bmc/goto_bmc_parse_options.cpp:52` which has a call to `is_goto_binary`, defined under `src/goto-programs/read_goto_binary.cpp`

Reply:
If you need it then the API needs to expose it, no?

Reply:
I'm wrong, we already expose it in the API - we will change it to make it defer to the API (we already have this capability, probably lost the idea somewhere during refactoring)